src/React.purs

```diff
@@ -388,37 +342,6 @@ foreign import transformState :: forall props state eff.
   (state -> state) ->
   Eff (state :: ReactState ReadWrite | eff) Unit
 
--- | Create a React class from a specification.
-foreign import createClass' :: forall props state render eff.
-  Fn2
-    (forall a. Maybe a -> Nullable a)
-    (ReactSpec props state render eff)
-    (ReactClass props)
-
-createClass :: forall props state render eff.
-  ReactSpec props state render eff -> ReactClass props
-createClass spc = runFn2 createClass' toNullable spc
-
--- | Create a stateless React class.  When using a non anonymous function the
--- | displayName will be the capitalized name of the function, e.g.
--- | ``` purescript
--- | helloWorld = createClassStatelesss helloWorldCls
--- |    where
--- |      helloWorldCls props = ...
--- | ```
--- | Then the `displayName` will be set up to `HelloWorldCls`
-foreign import createClassStateless :: forall props render.
-  ReactRender render =>
-  (props -> render) -> ReactClass props
```

Comment:
I don't know why these functions exist since this can be implemented with PureScript functions. If we do want to add this back, then they can be implemented in PureScript on top of `pureComponent`.